nodejs/FatBeaconPeripheral/README.md

```diff
@@ -0,0 +1,30 @@
+This is a nodejs implementation of a Fatbeacon developed on a Raspberry Pi 3.
```

Comment:
I'm trying to keep track of the tasks I've finished. Do you get an email everytime I comment
"done"
on one of your comments? If so, I can stop